### ISSUE TITLE
Case for Configuration variables

### ISSUE THREAD

**Issue by dkav** [linkagescape/linkage-mapper#8]
Configuration variables are named using lowercase characters in [cc_config.py](linkage-mapper/toolbox/scripts/cc_config.py) and uppercase characters in [lm_config.py](linkage-mapper/toolbox/scripts/lm_config.py). Lowercase is probably the more accurate approach for variables defined at runtime by tool input parameters.

**Comment by johngallo** [comment]
If I understand correctly, this still works, but is inconsistent...

**Comment by dkav** [comment]
Yes. Either approach could be taken. It is a common convention in Python to represent constants in uppercase. However, unlike other languages, true constants do not exist in Python. Some people regard uppercase naming as 'shouty'.

The question to be answered is - Do we view configuration variables that do not change as constants, even though they are defined at runtime? 

BTW, I haven't checked the other configuration files.
